<?xml version="1.0"?>
<launch>
  <!-- implement this to make things easier for later -->
  <arg name="params_file" default="ur5e_workstation_sim_params" />

  <include file="$(find uml_hri_nerve_pick_and_place)/launch/gpd_ur5.launch" />

  <include file="$(find armada_gazebo)/universal_robot/launch/gazebo_universal_robot.launch" >
    <arg name="robot_make" value="universal_robot" />
    <arg name="robot_model" value="ur5e"/>
    <arg name="sim_workstation" value="true"/>
    <arg name="gripper_make" value="robotiq" />
    <arg name="gripper_model" value="r2f85" />
    <arg name="wrist_cam" value="d435_millibar"/>  <!-- d435_millibar -->
  </include>

  <rosparam file="$(find uml_hri_nerve_pick_and_place)/cfg/ur5e_workstation_sim_params.yaml" />

  <node name="pick_and_place_gazebo" pkg="uml_hri_nerve_pick_and_place" type="pick_and_place_gazebo" output="screen"/>
  <node name="spawn_object" pkg="uml_hri_nerve_pick_and_place" type="spawn_object.py" output="screen"/>

</launch>
